build,packages: Restrict runc version

gdemonet opened the pull request:
The runc version 1.1.3, recently added in upstream repositories for
RHEL/Rocky 8, seems to include a bug causing issues with "exec" in
containers.
We explicitly reject this version in the RPM `Requires` line, hoping
that future versions will have this issue fixed.

See: https://github.com/containerd/containerd/issues/7219

gdemonet committed:
build,packages: Restrict runc version

The runc version 1.1.3, recently added in upstream repositories for
RHEL/Rocky 8, seems to include a bug causing issues with "exec" in
containers.
We explicitly reject this version in the RPM `Requires` line, hoping
that future versions will have this issue fixed.

See: https://github.com/containerd/containerd/issues/7219

bert-e commented:
# Hello gdemonet,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/metalk8s/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @nicolast

* @lucieleonard

* @teddyandrieux

* @chengyanjin

* @gdemonet

* @jbwatenbergscality

* @thomasdanan

* @alexandre-allard-scality

gdemonet committed:
salt: Do not explicitly install runc

This package is already required by containerd, and we don't need it
otherwise.

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/prevent-runc-1.1.3` into
`development/123.0`
* [w/124.0/bugfix/prevent-runc-1.1.3](https://github.com/scality/metalk8s/compare/development/124.0...w/124.0/bugfix/prevent-runc-1.1.3) will be merged into `development/124.0` 



The following branches will **NOT** be impacted:

* `development/2.0`
* `development/2.1`
* `development/2.10`
* `development/2.11`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`
* `development/2.9`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @nicolast

* @lucieleonard

* @teddyandrieux

* @chengyanjin

* @gdemonet

* @jbwatenbergscality

* @thomasdanan

* @alexandre-allard-scality

bert-e commented:
# History mismatch



Merge commit #a43ada68b6f3a3076533c8423c150d5e85c216b4 on the integration branch
`w/124.0/bugfix/prevent-runc-1.1.3` is merging a branch which is neither the current
branch `bugfix/prevent-runc-1.1.3` nor the development branch
`development/124.0`.

It is likely due to a rebase of the branch `bugfix/prevent-runc-1.1.3` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**

gdemonet commented:
/reset

bert-e commented:
# Reset complete



I have successfully deleted this pull request's integration branches.

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/prevent-runc-1.1.3` into
`development/123.0`
* [w/124.0/bugfix/prevent-runc-1.1.3](https://github.com/scality/metalk8s/compare/development/124.0...w/124.0/bugfix/prevent-runc-1.1.3) will be merged into `development/124.0` 



The following branches will **NOT** be impacted:

* `development/2.0`
* `development/2.1`
* `development/2.10`
* `development/2.11`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`
* `development/2.9`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @nicolast

* @lucieleonard

* @teddyandrieux

* @chengyanjin

* @gdemonet

* @jbwatenbergscality

* @thomasdanan

* @alexandre-allard-scality

TeddyAndrieux reviewed (dismissed):
LGTM

bert-e commented:
# History mismatch



Merge commit #bb3d0b7f5890347b3b81ba1236e409200bb9d995 on the integration branch
`w/124.0/bugfix/prevent-runc-1.1.3` is merging a branch which is neither the current
branch `bugfix/prevent-runc-1.1.3` nor the development branch
`development/124.0`.

It is likely due to a rebase of the branch `bugfix/prevent-runc-1.1.3` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**

TeddyAndrieux reviewed (approved):
LGTM - again :smile:

TeddyAndrieux commented:
/reset

bert-e commented:
# Reset complete



I have successfully deleted this pull request's integration branches.

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/prevent-runc-1.1.3` into
`development/123.0`
* [w/124.0/bugfix/prevent-runc-1.1.3](https://github.com/scality/metalk8s/compare/development/124.0...w/124.0/bugfix/prevent-runc-1.1.3) will be merged into `development/124.0` 



The following branches will **NOT** be impacted:

* `development/2.0`
* `development/2.1`
* `development/2.10`
* `development/2.11`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`
* `development/2.9`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @nicolast

* @lucieleonard

* @teddyandrieux

* @chengyanjin

* @gdemonet

* @jbwatenbergscality

* @thomasdanan

* @alexandre-allard-scality

gdemonet commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/123.0`

* :heavy_check_mark: `development/124.0`



The following branches will **NOT** be impacted:

* `development/2.0`
* `development/2.1`
* `development/2.10`
* `development/2.11`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`
* `development/2.9`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/123.0`

* :heavy_check_mark: `development/124.0`



The following branches have **NOT** changed:

* `development/2.0`
* `development/2.1`
* `development/2.10`
* `development/2.11`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`
* `development/2.9`



Please check the status of the associated issue None.

Goodbye gdemonet.
